packages/core/src/components/dialog/_dialog-footer.scss

```diff
@@ -0,0 +1,40 @@
+.#{$ns}-dialog-footer {
+  flex: 0 0 auto;
+}
+
+.#{$ns}-dialog-fixed-footer {
+  background-color: $white;
+  border-radius: 0  0 $dialog-border-radius $dialog-border-radius;
+  border-top: 1px solid $pt-divider-black;
+  padding: $pt-grid-size;
+  padding-left: $dialog-padding;
```

Comment:
why the extra padding here? is this intentional? if so, should we add the extra (5px) padding on the right side as well?

| before | after |
| - | - |
| <img width="853" alt="image" src="https://user-images.githubusercontent.com/723999/211428484-b68913de-e166-45bd-b35a-3ae909565d65.png"> | <img width="865" alt="image" src="https://user-images.githubusercontent.com/723999/211428503-5c8a5e23-64f1-4d14-b887-7973404af906.png"> |

Reply:
I see now that it's a little tricky to design this style to work for both button and non-button content. In the docs example we have some text in the "left" section for Dialog, but there's a button in that same place for MultiStepDialog...